core/settings/qPrefCloudStorage.h

```diff
@@ -30,19 +31,19 @@ class qPrefCloudStorage : public QObject {
 	void sync() { loadSync(true); }
 
 public:
-	const QString cloud_base_url() const;
-	const QString cloud_git_url() const;
-	const QString cloud_storage_email() const;
-	const QString cloud_storage_email_encoded() const;
-	const QString cloud_storage_newpassword() const;
-	const QString cloud_storage_password() const;
-	const QString cloud_storage_pin() const;
-	int   cloud_timeout() const;
-	int   cloud_verification_status() const;
-	bool  git_local_only() const;
-	bool  save_password_local() const;
-	bool  save_userid_local() const;
-	const QString userid() const;
+	static inline const QString cloud_base_url() { return prefs.cloud_base_url; };
```

Comment:
Lots of unnecessary semicolons...

Reply:
corrected, but remark it is not an error to add ; for many it is a natural way to close a statement (just like it mandatory in class definitions)

Reply:
> corrected, but remark it is not an error to add ; for many it is a natural way to close a statement (just like it mandatory in class definitions)

We discussed this before. :) There is *no* semicolon at the end of a function definition. You're simply adding random ';' to your class. The compiler will happily compile this:
```
class A {
    ;;;;;;;;;
};
```
I'm not sure if you are aware, but this is likewise valid:
```
void f()
{
};
```

Reply:
i am aware of many ways to misuse the compiler, I have wriiten enough backends (p-code to assembler) to see that, and I know we discussed it before.

I am aware you try to remove every extra not strictly needed term, I look at it differently, if it does not generate extra code why not make the code easier to read (think of your automatic conversions as well).

However do note that the code is changed to match your style.